dor/builders/parts.py

```diff
@@ -182,6 +182,12 @@ def ext(self):
                 return "tiff"
             case "text/plain":
                 return "txt"
+            case "text/plain; charset=US-ASCII":
+                return "txt"
+            case "text/plain; charset=UTF-8":
+                return "txt"
```

Comment:
These weren't really needed in this PR, so I could remove them. **I think** @gkostin1966 may need them for his transcript work though, where a text file `FileInfo` will be created with the mimetype from JHOVE.